alpha/apps/kaltura/lib/myEntryUtils.class.php

```diff
@@ -1195,8 +1209,8 @@ public static function getEntryDataPath($flavorSyncKey, $flavorAsset, $entryId)
 			$isCloudDc = myCloudUtils::isCloudDc($currentDcId);
 			if ($isCloudDc && (in_array($fileSync->getDc(), kStorageExporter::getPeriodicStorageIds())))
 			{
-				KalturaLog::info('Current DC id: ' . $currentDcId . ' is cloud DC');
-				$entryDataPath = $fileSync->getExternalUrl($entryId, null, true);
+				KalturaLog::info("Current DC id: $currentDcId is cloud DC $isCloudDc]");
+				$entryDataPath = $fileSync->getFullPath();
```

Comment:
Since we now also use  $fileSync->getFullPath(); for this case, maybe its worth changing the function to only have 2 cases id($isCloudDc || $fileSync->getDc() === $currentDcId) do $fileSync->getFullPath(); else dump Api request

Reply:
Agree, no reason to have the 3 if-else, minimized to 2.